src/simulation/dynamics/spinningBodies/spinningBodiesOneDOF/spinningBodyOneDOFStateEffector.cpp

```diff
@@ -39,17 +39,17 @@ SpinningBodyOneDOFStateEffector::SpinningBodyOneDOFStateEffector()
     this->r_SB_B.setZero();
     this->sHat_S.setZero();
 
-    this->nameOfThetaState = "spinningBodyTheta" + std::to_string(this->effectorID);
-    this->nameOfThetaDotState = "spinningBodyThetaDot" + std::to_string(this->effectorID);
-    this->effectorID++; 
+    this->nameOfThetaState = "spinningBodyTheta" + std::to_string(SpinningBodyOneDOFStateEffector::effectorID);
+    this->nameOfThetaDotState = "spinningBodyThetaDot" + std::to_string(SpinningBodyOneDOFStateEffector::effectorID);
+    SpinningBodyOneDOFStateEffector::effectorID++;
 }
 
 uint64_t SpinningBodyOneDOFStateEffector::effectorID = 1;
 
 /*! This is the destructor, nothing to report here */
 SpinningBodyOneDOFStateEffector::~SpinningBodyOneDOFStateEffector()
 {
-    this->effectorID = 1;    /* reset the panel ID*/
+    SpinningBodyOneDOFStateEffector::effectorID = 1;    /* reset the panel ID*/
```

Comment:
:small_red_triangle_down: Remove the comment on this line.

Reply:
Done.